test/files/neg/names-defaults-neg.check

```diff
@@ -45,26 +45,26 @@ names-defaults-neg.scala:28: error: Int does not take parameters
   test3(b = 3, a = 1)(3)
                      ^
 names-defaults-neg.scala:37: error: ambiguous reference to overloaded definition,
-both method f in object t1 of type (b: String, a: Int)String
-and  method f in object t1 of type (a: Int, b: String)String
+both method f in object t1 of type (b: String, a: Int): String
+and  method f in object t1 of type (a: Int, b: String): String
 match argument types (b: String,a: Int)
   t1.f(b = "dkljf", a = 1)
      ^
 names-defaults-neg.scala:44: error: ambiguous reference to overloaded definition,
-both method f in object t3 of type (a2: Int)(b: Int)String
-and  method f in object t3 of type (a1: Int)String
+both method f in object t3 of type (a2: Int): (b: Int): String
```

Comment:
Does this format of multiple parameter lists look as intended? `(a2: Int): (b: Int): String` looks as strange as `(a2: Int)(b: Int)String` did.